mswindows/osgeo4w/env.bat.tmpl

```diff
@@ -11,19 +11,21 @@ REM Note that msys package must be also installed
 REM set GRASS_SH=%OSGEO4W_ROOT%\apps\msys\bin\sh.exe
 
 set GRASS_PYTHON=%OSGEO4W_ROOT%\bin\python3.exe
-set PYTHONHOME=%OSGEO4W_ROOT%\apps\Python37

```

Comment:
yes, it's defined in `C:\OSGeo4W64\bin\py3_env.bat`

`SET PYTHONHOME=%OSGEO4W_ROOT%\apps\Python37`